posthog/api/team.py

```diff
@@ -311,6 +313,31 @@ def validate_session_recording_network_payload_capture_config(value) -> dict | N
 
         return value
 
+    @staticmethod
+    def validate_session_recording_masking_config(value) -> dict | None:
+        if value is None:
+            return None
+
+        if not isinstance(value, dict):
+            raise exceptions.ValidationError("Must provide a dictionary or None.")
+
+        allowed_keys = {"maskAllInputs", "maskTextSelector"}
+
+        if not all(key in allowed_keys for key in value.keys()):
+            raise exceptions.ValidationError(
+                f"Must provide a dictionary with only known keys: {', '.join(allowed_keys)}."
+            )
+
+        if "maskAllInputs" in value:
+            if not isinstance(value["maskAllInputs"], bool):
+                raise exceptions.ValidationError("maskAllInputs must be a boolean.")
+
+        if "maskTextSelector" in value:
+            if not isinstance(value["maskTextSelector"], str):
```

Comment:
do we accept arbitrary strings 🤔 
i think safe to allow any string but not present that in the UI otherwise we can be more strict here